support/scheduler-client/client.go

```diff
@@ -0,0 +1,101 @@
+/*******************************************************************************
+ * Copyright 2017 Tencent Inc.
```

Comment:
Consider using SPDX-style comments (https://spdx.org/) to improve readability and reduce comment bloating: https://github.com/edgexfoundry/edgex-go/blob/master/export/client/server.go#L1-L5